docs/datasets/index.rst

```diff
@@ -52,10 +52,106 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
 energy bin, e.g. for modeling and fitting,
 see the `2D map analysis tutorial <./tutorials/image_analysis.html>`__.
 
-
-Getting Started
-===============
-
+To analyse multiple runs, you can either stack the datasets together, or perform
+a joint fit across multiple datasets.
+
+.. _stack:
+
+Stacking Multiple Datasets
+==========================
+
+Stacking datasets implies that the counts, background and reduced IRFs from all the
+runs are binned together to get one final dataset for which a likelihood is
+computed during the fit. Stacking is often useful to reduce the computation effort while
+analysing multiple runs.
+
+Counts, background and exposure (in the data range defined by the `mask_safe`)
+are simply summed,
+while for energy dispersion and point spread function an average is computed after
+weighing by the exposure.
+
+For the model evaluation, an important factor that needs to be accounted for is
+that the energy threshold changes between obseravtions.
+To ensure that the `npred` (ie, the predicted number of counts) on the stacked
+dataset is the sum expected by stacking the npred of the individual runs,
+a `~gammapy.irf.EDispersionMap` is used, which contains the
+The mask_safe from each dataset is applied on the respective reconstructed energy axis
+of the energy dispersion matrix, and the masked matrices are combined.
+Values lying outside the safe mask of each dataset are lost.
+
+The following plot shows the individual and stacked attributes for two `SpectrumDataset`
+
+.. plot:: datasets/plot_stack.py
```

Comment:
From my perspective I think a plot just showing that npred is conserved would be sufficient. I think just adding two spectra as done for counts / background is maybe too simple to illustrate...

Reply:
We can remove it if you want, but generally among beginners, there is some confusion as to what stacking really means - showing an example with counts and background can perhaps be helpful, and that npred stacking works really the same way